webpack/assets/javascripts/react_app/components/Layout/NavigationSearch.js

```diff
@@ -0,0 +1,231 @@
+      } else if (
+        !isAutocompleteOpen &&
+        searchInputRef?.current &&
+        searchInputRef.current.contains(event.target)
+      ) {
+        // The autocomplete menu should open if the user clicks on the search input.
+        setIsAutocompleteOpen(true);
+      }
+    };
+    const handleMenuKeys = event => {
+      if (
+        isAutocompleteOpen &&
+        searchInputRef.current &&
+        searchInputRef.current === event.target
+      ) {
+        // the escape key closes the autocomplete menu and keeps the focus on the search input.
+        if (event.key === 'Escape') {
+          setIsAutocompleteOpen(false);
+          searchInputRef.current.focus();
+          // the up and down arrow keys move browser focus into the autocomplete menu
+        } else if (event.key === 'ArrowDown' || event.key === 'ArrowUp') {
+          const firstElement = autocompleteRef.current.querySelector(
+            'li > a:not(:disabled)'
+          );
+          firstElement && firstElement.focus();
+          event.preventDefault(); // by default, the up and down arrow keys scroll the window
+          // the tab, enter, and space keys will close the menu, and the tab key will move browser
+          // focus forward one element (by default)
+        } else if (
+          event.key === 'Tab' ||
+          event.key === 'Enter' ||
+          event.key === 'Space'
+        ) {
+          setIsAutocompleteOpen(false);
+          if (event.key === 'Enter' || event.key === 'Space') {
+            event.preventDefault();
+          }
+        }
+        // If the autocomplete is open and the browser focus is in the autocomplete menu
+        // hitting tab will close the autocomplete and but browser focus back on the search input.
+      } else if (
+        isAutocompleteOpen &&
+        autocompleteRef?.current?.contains(event.target) &&
+        event.key === 'Tab'
+      ) {
+        event.preventDefault();
+        setIsAutocompleteOpen(false);
+        searchInputRef.current.focus();
+      }
+    };
+    window.addEventListener('keydown', handleMenuKeys);
+    window.addEventListener('click', handleClick);
+    return () => {
+      window.removeEventListener('keydown', handleMenuKeys);
+      window.removeEventListener('click', handleClick);
+    };
+  }, [isAutocompleteOpen]);
+  useEffect(() => {
+    const handleKeyDown = event => {
+      if (event.ctrlKey && event.shiftKey && event.key === 'F') {
+        // Focus the search input on Ctrl+Shift+F
+        if (searchInputRef.current) {
+          searchInputRef.current.focus();
+        }
+      }
+    };
+
+    // Add the event listener
+    window.addEventListener('keydown', handleKeyDown);
+
+    // Remove the event listener when the component unmounts
+    return () => {
+      window.removeEventListener('keydown', handleKeyDown);
+    };
+  }, []);
+
+  const searchInput = (
+    <SearchInput
+      value={value}
+      placeholder={__('Search and go')}
```

Comment:
Could we change it to `Ctrl + Shift + F to search`? I mean, how would one know what to use?

Not a big deal though, since that combination is used in a lot of editors...

Reply:
I talked to Maria from UX about this, and since we (and other projects) dont show the `/` shortcut in the search bar placeholder we shouldn't do it here as well. I'll get it in the docs and I added it to the about page 
![image](https://github.com/theforeman/foreman/assets/30431079/0dbc8714-2a8e-4da8-a72f-3b94b69c18e7)